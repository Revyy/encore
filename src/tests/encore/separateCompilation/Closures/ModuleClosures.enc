import Libs.Closures
import Libs.Greetings


active class Main
  def main() : unit
    val callr = new ClosureCaller
    println(callr.start(GreetSwedish))
    println(callr.start(GreetEnglish))
    println(callr.start(GreetEnglishFancy))
  end
end

