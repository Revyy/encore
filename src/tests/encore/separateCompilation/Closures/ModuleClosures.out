Hej
Hello
Good day to you sir!